README.md

```diff
@@ -33,29 +33,49 @@ For more information about getting started with git, please check out the
 
 ## Configuration
 
-Specify your [API Key, site subdomain](https://app.recurly.com/go/developer/api_access), and (optionally) page size
-setting in your `app.config` or `web.config` file:
+To configure the library, you will need your [API Key and site subdomain](https://app.recurly.com/go/developer/api_access).
+The recommended way to configure the library is to statically initialize it:
 
-```xml
-<?xml version="1.0" encoding="utf-8" ?>
-<configuration>
-  <configSections>
-    <section name="recurly" type="Recurly.Configuration.Section,Recurly"/>
-  </configSections>
-
-  <recurly
-    apiKey="123456789012345678901234567890ab"
-    subdomain="company"
-	pageSize="50" /> <!-- optional. 50 is the default -->
-
-</configuration>
+```csharp
+// pageSize is optional and defaults to 200
+Recurly.Configuration.SettingsManager.Initialize(apiKey, subdomain, pageSize);
 ```
 
+We recommend doing this for compatibility with newer .NET frameworks as well as security. We also recommend you encrypt
+your api key at rest or use a secrets service. *You should not store your api key in plain text on your system.*
+
+#### Configuration Security Note
+
+If you are using the legacy method of configuring with `app.config` or `web.config`, we recommend updating your
+application to the method recommend above.
+
 ## Client Documentation
 
 The API documentation is available on our [developer docs site](https://dev.recurly.com/docs/getting-started)
 You can find .NET examples there but we have some examples in [examples.md](./examples.md).
 
+## .NET core and standard support
+
+Although we do not officially support .NET core or standard at this moment, it is possible to use this library
+in those applications. We recommend vendoring the code, building the dlls, and linking directly to them from
```

Comment:
I don't think "recommend" is the right word here. How about "It is possible to vendor the code..."